app/src/main/res/layout/fraction_interaction_item.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+  <data>
+    <variable
+      name="viewModel"
+      type="org.oppia.app.player.state.itemviewmodel.FractionInteractionViewModel"/>
+  </data>
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+    <FrameLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_margin="8dp"
+      android:background="@color/white"
+      android:orientation="vertical"
+      android:padding="8dp"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintTop_toTopOf="parent">
+      <org.oppia.app.customview.interaction.FractionInputInteractionView
```

Comment:
This view needs to be restricted with digits `android:digits="0123456789/- "`. If we enter any string other than number it crashes.

Reply:
Oh! Interesting. For some reason when I tried this before, it didn't seem to filter out the extra characters but it's working now. Re-added--thanks for pointing this out to me.